package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "chrome-extension",
```

Not necessary here, but I like to add a scope to our packages
```suggestion
  "name": "@evolv/chrome-extension",
```